phoenix-core/src/it/java/org/apache/phoenix/end2end/index/PartialIndexIT.java

```diff
@@ -603,6 +606,120 @@ public void testPhoenixRowTimestamp() throws Exception {
         }
     }
 
+    @Test
+    public void testViewIndexes() throws Exception {
+        String baseTableName =  generateUniqueName();
+        String globalViewName = generateUniqueName();
+        String globalViewIndexName =  generateUniqueName();
+        String tenantViewName =  generateUniqueName();
+        try (Connection conn = DriverManager.getConnection(getUrl())) {
+            conn.createStatement().execute("CREATE TABLE " + baseTableName +
+                    " (TENANT_ID CHAR(15) NOT NULL, PK2 DATE NOT NULL, PK3 INTEGER NOT NULL, " +
+                    "KV1 VARCHAR, KV2 VARCHAR, KV3 CHAR(15) " +
+                    "CONSTRAINT PK PRIMARY KEY(TENANT_ID, PK2, PK3)) MULTI_TENANT=true" +
+                    (salted ? ", SALT_BUCKETS=4" : ""));
+            conn.createStatement().execute("CREATE VIEW " + globalViewName +
+                    " AS SELECT * FROM " + baseTableName);
```

Comment:
You can repro the exception "java.sql.SQLException: ERROR 1046 (43A03): Column updated in VIEW may not differ from value specified in WHERE clause. value='a' columnName=KP" by adding a WHERE clause to the global view

Reply:
Thank you @jpisaac for finding this test failure. I fixed it.